qa/os/src/test/java/org/elasticsearch/packaging/util/Docker.java

```diff
@@ -506,11 +503,11 @@ private static void verifyDefaultInstallation(Installation es) {
             "x-pack-env",
             "x-pack-security-env",
             "x-pack-watcher-env"
-        ).forEach(executable -> assertPermissionsAndOwnership(es.bin(executable), p755));
+        ).forEach(executable -> assertPermissionsAndOwnership(es.bin(executable), p555));
```

Comment:
If the platform difference doesn't make it too complicated you could maybe add a similar assertion for the ML programs: `controller`, `autodetect`, `normalize` and `data_frame_analyzer`.

They're in `$ES_HOME/modules/x-pack-ml/platform/linux-x86_64/bin` on `x86_64` and `$ES_HOME/modules/x-pack-ml/platform/linux-aarch64/bin` on `aarch64`.

Reply:
I added a check for the ML binaries, see what you think.

Reply:
Thanks.  `pytorch_inference` will need to be removed from that list in the backport to 7.x, but looks good.